methods/build.js

```diff
@@ -38,6 +38,9 @@ module.exports = function (config, settings, data) {
     if (item.config.monorepo && item.config.monorepoHook) {
       envVars.WEBHOOK_MONOREPO = item.config.monorepoHook.urls;
       envVars.WEBHOOK_DATA = qs.stringify(item.config.monorepoHook.payload);
+      if (settings.runDockerBuilder) {
```

Comment:
Should this be configurable? Or should it run all the time?

Reply:
all the time is fine